gr-util/src/grdeviceaddon.cpp

```diff
@@ -27,14 +27,15 @@ QWidget *GRDeviceAddon::createAttrMenu(QWidget *parent)
 	MenuSectionWidget *attrContainer = new MenuSectionWidget(parent);
 	MenuCollapseSection *attr =
 		new MenuCollapseSection("ATTRIBUTES", MenuCollapseSection::MHCW_NONE, attrContainer);
-	QList<IIOWidget *> attrWidgets = IIOWidgetBuilder().device(m_src->iioDev()).buildAll();
+	QList<IIOWidget *> attrWidgets = IIOWidgetBuilder().device(m_src->iioDev()).parent(parent).buildAll();
```

Comment:
Refactor to IIOWidgetBuilder(QWidget *parent=nullptr) and remove .parent(QWidget) setting